package.json

```diff
@@ -22,6 +22,7 @@
     "@tanstack/react-table": "8.20.5",
     "chroma-js": "3.1.1",
     "clsx": "2.1.1",
+    "lucide-react": "0.452.0",
```

Comment:
Please use [react-icons](https://react-icons.github.io/react-icons/)

Reply:
Fixed, please check now

Reply:
There's no change in `package.json`, so even `package-lock.json` should be removed

Reply:
Check now bro

Reply:
Thanks